misc/formatter/modules/formatter-core/src/test/resources/declarations/class-definition/assert/class_definition_declaration_17.bal

```diff
@@ -0,0 +1,11 @@
+class Foo {
+    public function init() {
+    }
+
+    function getName() {
+    }
```

Comment:
Latest changes keeps a blank link in between empty blocks.

Reply:
Updated.